DependencyInjection/Configuration.php

```diff
@@ -72,6 +72,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
+                ->booleanNode('toolbar')->defaultTrue()->end()
```

Comment:
i would make toolbar an arrayNode with canBeEnabled and a message_formatter option to use a custom formatter. its not a plugin configuration for me. the fact that the history plugin is used is an implementation detail. if i configure my own services with history plugin instances, i need my own configuration for formatters anyways.

the other plugins can be configured and then used as they are in any client the user wants to use them.